### PR TITLE
Update to use standard 7.1.2

### PR DESCRIPTION
**Pull request by musgravejw**
Closes:  https://github.com/onmodulus/rabbit-pubsub/issues/3

### CODE REVIEW

**Comment by tzmanics** [comment]
I would just make a note that you've also properly re-linted and added missing `tape` dependency. Once you figure out what's going on with travis 👍

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "url": "https://github.com/onmodulus/rabbit-pubsub/issues"
   },
   "devDependencies": {
-    "@modulus/standard": "0.6.0",
+    "standard": "7.1.2",
```
Comment (franvarney) [inline package.json:10]: Why did you change this?

Comment (musgravejw) [inline package.json]: Oh, I believe this was also changed in `master`, could be a bad rebase.

Comment (franvarney) [inline package.json:10]: I don't understand. It's always been `@modulus/standard` and should continue to be, yeah...?

Comment (musgravejw) [inline package.json:10]: I'm fine with using `@modulus/standard`.

**Comment by theworkflow** [comment]
Will this PR go from using `@modulus/standard` to `standard`? I'm not sure if you'll be changing that back and updating the version of `mod-standard`.

**Review by jackboberg — APPROVED** [review]

**Comment by musgravejw** [comment]
Addressed further here:  https://github.com/onmodulus/rabbit-pubsub/pull/8
